public/javascripts/SVLabel/src/SVLabel/Keyboard.js

```diff
@@ -30,13 +32,76 @@ function Keyboard ($) {
         $(document).bind('keyup', documentKeyUp);
         $(document).bind('keydown', documentKeyDown);
     }
+     // taken from 
+    // http://stackoverflow.com/questions/7799722/how-do-i-implement-move-forward-action-for-google-street-view
+    function headingDifference(link, angle) {
+    
+        var diff = Math.abs((svl.panorama.pov.heading % 360) - link.heading + angle);
+        if(diff>180)
+            diff=Math.abs(360-diff);
+
+        return diff;
+
+    }
+
+    function movePano(angle) {
+        var curr;
+        var found = false;
+        var len =  svl.panorama.links.length;
+        for(var i = 0; i < len; i++) {
+            var diffLast = headingDifference(svl.panorama.links[i], angle);
+            if(curr == undefined) {
+                curr = svl.panorama.links[i];
+            }
+            if(diffLast <= 90){
+                found = true;
+            }
+            var diffCurr = headingDifference(curr, angle);
+            if(diffCurr > diffLast && diffLast <= 90) {
+                curr = svl.panorama.links[i];
+                found = true;
+            }
+        }
+        if(found){
+            svl.panorama.setPano(curr.pano);
+        }
+    }
+
+    function rotatePov(degree){
+        var heading =  svl.panorama.pov.heading;
+        var pitch = svl.panorama.pov.pitch;
+        var zoom = svl.panorama.pov.zoom;
+        heading += degree;
+        if(heading < 0)
+            heading += 360;
+        heading = heading % 360;
+        svl.panorama.setPov({heading: heading, pitch: pitch, zoom: zoom});
+
+    }
 
     /**
      * This is a callback for a key down event
      * @param {object} e An event object
      * @private
      */
     function documentKeyDown(e) {
+        if([32, 37, 38, 39, 40].indexOf(e.keyCode) > -1) { 
+            e.preventDefault();
+        }
+    
+        switch (e.keyCode) {
+            // "left"
+            // rotate pano
+            case 37:
+                rotatePov(-2);
```

Comment:
Originally tried with +/-1 which was fine but a little slow. The rate of the rotation depends on the browser's refresh rate.